hal_st_lwip/instantiations_lwip/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
-add_library(hal_st.instantiations_lwip STATIC)
-if (EMIL_HOST_BUILD OR TARGET_MCU_FAMILY STREQUAL stm32wbxx)
-    set_property(TARGET hal_st.instantiations_lwip PROPERTY EXCLUDE_FROM_ALL TRUE)
+add_library(hal_st.instantiations_lwip ${HALST_EXCLUDE_FROM_ALL} STATIC)
+if (EMIL_HOST_BUILD OR "${TARGET_MCU_FAMILY}" STREQUAL stm32wbxx)
+    set_property(TARGET hal_st.instantiations_lwip PROPERTY EXCLUDE_FROM_ALL On)
```

Comment:
Please keep On/Off for configurable options, and TRUE/FALSE for boolean logic of properties

Reply:
```suggestion
    set_property(TARGET hal_st.instantiations_lwip PROPERTY EXCLUDE_FROM_ALL TRUE)
```